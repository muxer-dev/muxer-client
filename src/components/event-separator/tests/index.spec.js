import React from 'react';
import EventSeparator from '../';

describe('event-separator', function () {
  it('renders', function () {
    expect(<EventSeparator id='test-id' content='Test Content' />).toMatchSnapshot();
  });
});
