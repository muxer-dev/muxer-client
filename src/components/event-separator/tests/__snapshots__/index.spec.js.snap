// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`event-separator renders 1`] = `
<EventSeparator
  className=""
  content="Test Content"
  id="test-id"
/>
`;
